packages/generator-honeycomb/__tests__/app/index.test.js

```diff
@@ -217,60 +194,44 @@ describe('generator-honeycomb', () => {
           it('should have expected content', () => {
             assert.fileContent([
               ['package.json', '"build": "npm run build:babel && npm run build:webpack",'],
-              ['package.json', '"clean": "rimraf pids logs coverage dist output public/javascripts/*.bundle.js"'],
-              ['package.json', '"lint:styles": "stylelint src/server/views/**/*.js src/client/components/**/*.js",'],
               ['package.json', '"start:development": "cross-env NODE_ENV=development nodemon src/server/server.js --exec babel-node --watch src/server --ext js",'],
-              ['package.json', /"hapi-react-views": ".*",/],
-              ['package.json', /"react": ".*",/],
-              ['package.json', /"react-dom": ".*",/],
-              ['package.json', /"react-hot-loader": ".*",/],
-              ['package.json', /"babel-preset-react": ".*",/],
+              ['package.json', /"hapi-preact-views": ".*",/],
+              ['package.json', /"preact": ".*",/],
+              ['package.json', /"babel-plugin-transform-react-jsx": ".*",/],
               ['package.json', /"eslint-plugin-jsx-a11y": ".*",/],
               ['package.json', /"eslint-plugin-react": ".*",/],
-              ['package.json', /"react-test-renderer": ".*",/],
-              ['package.json', /"styled-components": ".*",/],
-              ['package.json', /"stylelint-processor-styled-components": ".*"/],
+              ['package.json', /"preact-render-to-string": ".*",/],
             ]);
           });
         });
 
         describe('.eslintrc.yml', () => {
           it('should have expected content', () => {
-            assert.fileContent('.eslintrc.yml', 'honeycomb/react');
-          });
-        });
-
-        describe('.stylelintrc.yml', () => {
```

Comment:
I think we can remove this test, because it's the same for each engine:

```js
describe('.stylelintrc.yml', () => {
  it('should have expected content', () => {
    assert.fileContent('.stylelintrc.yml', '"stylelint-config-standard"');
  });
});
```